model-optimizer/extensions/front/tf/TFSliceToSlice.py

```diff
@@ -76,6 +76,8 @@ def replace_sub_graph(self, graph: Graph, match: dict):
         select_node.out_port(0).connect(slice_node.in_port(2))
 
         cast = Cast(graph, dict(name=sum_node.name + '/CastToI64', dst_type=np.int64)).create_node()
+        cast_int32max = Cast(graph, dict(name=sum_node.name + '/CastToI64', dst_type=np.int64)).create_node()
```

Comment:
If you have already generated IR I suggest to remove these lines so that they will not conflict with changes https://github.com/openvinotoolkit/openvino/pull/4367 that fix such king of issues for the future

Reply:
Done.